langchain4j-core/src/main/java/dev/langchain4j/model/audio/AudioModel.java

```diff
@@ -0,0 +1,14 @@
+package dev.langchain4j.model.audio;
+
+import dev.langchain4j.data.audio.Audio;
+import dev.langchain4j.model.output.Response;
+
+public interface AudioModel {
+    /**
+     * Given an audio file, generates a transcription.
+     * 
+     * @param audioFile The audio file to generate a transcription from.
+     * @return The generated transcription response.
+     */
+    Response<String> transcribe(Audio audioFile);
```

Comment:
```suggestion
    AudioTranscriptionResponse transcribe(AudioTranscriptionRequest request);
```

Let's make it evolvable from the very beginning.
Ofc we can also expose `String transcribe(Audio audioFile)` method for convenience.

Please do not use `Response<T>` class in new APIs, it will be deprecated soon.